src/__tests__/Form.steps.tsx

```diff
@@ -18,11 +18,11 @@ describe("Form", function () {
 			fireEvent.change(fieldElement, {target: {value: value}});
 		});
 
-	const xInputIsClicked = (defineStepFunction: DefineStepFunction) =>
+	const xInputTypeAndErase = (defineStepFunction: DefineStepFunction) =>
```

Comment:
l'ancienne méthode fonctionnait bien pour simuler le in/out du champ afin de déclencher l'erreur,
ça ne fonctionne pas avec react hook form ?

Reply:
Nope c'est pour ça j'ai du faire un ajustement